src/ts2fable.fs

```diff
@@ -42,7 +40,9 @@ let transform (file: FsFile): FsFile =
     |> removeDuplicateFunctions
 
 let writeFile (tsPaths: string list) (fsPath: string): unit =
-    let path = Fable.Import.Node.Exports.Path
+    // let path = Fable.Import.Node.Exports.Path
+    let path: Node.path.IExports = importAll "path"
+    let fs: Node.fs.IExports = importAll "fs"
```

Comment:
This looks good to me. Now I need to see about getting those in the Node module.